ethereum/Deploying.md

```diff
@@ -68,6 +68,19 @@ module.exports = async function (deployer) {
 }
 ```
 
+When changing the storage, you might need to disable the storage because Open Zeppelin is very conservative,
+and appending to the Pyth State struct is considered illegal. For that, you can add 
```

Comment:
Might be worth explaining exactly why this is an issue for us (i.e. because we store our state variables in a nested struct).

Reply:
Sure. Done.